metabase/metabase
# Add color palette preview

Epic https://github.com/metabase/metabase/issues/22816

Changes:
- Adds a preview in a form of sample chart in Admin settings.
- Adds support for smaller screens for color settings
- Cleans up `colors/groups` a bit

How to test:
- Run Metabase EE
- Go to Admin -> Settings -> Appearance
- Make sure the chart is displayed for default values
- Change colors and make sure the chart is updated

<img width="1104" alt="Screenshot 2022-06-01 at 21 41 19" src="https://user-images.githubusercontent.com/8542534/171478768-904c5e1a-716f-49a5-8592-316ddf696d73.png">

<img width="683" alt="Screenshot 2022-06-02 at 14 36 58" src="https://user-images.githubusercontent.com/8542534/171623030-51b69a35-99bf-4637-a9ec-e8209296fd2a.png">
